server/punishments.ts

```diff
@@ -874,7 +875,7 @@ export const Punishments = new class {
 		let affected: User[] = [];
 
 		if (!user || userId && userId !== user.id) {
-			affected = Punishments.roomPunishName(room, userId, punishment);
+			affected = Punishments.roomPunishName(room, userId as ID, punishment);
```

Comment:
```suggestion
			affected = Punishments.roomPunishName(room, userId, punishment);
```

Fix the `roomPunishName` signature, not here.

Reply:
You did the same thing; resolved this instead of fixing it.